src/main/java/study/StringUtil.java

```diff
@@ -0,0 +1,27 @@
+package study;
+
+public class StringUtil {
+    private StringUtil() {
+    }
+    private static final String INPUT_ERROR = "올바른 값을 입력해주세요.";
+
+    public static String[] split(String input, String divider) {
+        if (input.isEmpty() || divider.isEmpty() || !divider.equals(","))
+            throw new IllegalArgumentException(INPUT_ERROR);
```

Comment:
```suggestion
        if (input.isEmpty() || divider.isEmpty() || !divider.equals(",")) {
                throw new IllegalArgumentException(INPUT_ERROR);
        }
```

자바의 경우 한 줄의 if문에도 괄호를 사용해주는 것이 컨벤션입니다!

https://google.github.io/styleguide/javaguide.html 또는 https://myeonguni.tistory.com/1596를 참고해주세요 :)

Reply:
넵! :ok_woman: